_posts/2024-11-22-release-2.8.0.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: Release 2.8.0 of MDAnalysis
+---
+
+We are happy to release version 2.8.0 of MDAnalysis!
+
+This is a minor release of the MDAnalysis library, which means that it
+contains enhancements, bug fixes, deprecations, and other
+backwards-compatible changes.
+
+However, in this case *minor* does not quite do justice to what is
+happening in this release, given that we have (at least) three big
+changes/additions:
+
+1. The license was changed to the **Lesser General Public License** so
+   that MDAnalysis can be used by packages under *any* license while
+   keeping the source code itself free and protected.
+2. We introduce the [Guesser][guesser modules docs] API for
+   guessing missing topology attributes such as element or mass in a
+   *context-dependent* manner. Until release 3.0, you should not
+   notice any differences but under the hood we are getting ready to
+   make it easier to work with simulations in a different *context*
+   (e.g., with the MARTINI force field) or experimental PDB
+   files). With consistent attributes, such as elements, it becomes a
+   lot easier to interface with tools like the cheminformatics RDKit
+   (via the [converters][converter modules docs]). 
+   
+   The guessers are the [GSoC 2022 project][] of @aya9aladdin with
+   help from @lilyminium, @IAlibay, and @jbarnoud.[^guesserPR]
+3. We are introducing [parallel analysis][] for tools in
+   [MDAnalysis.analysis][] following the simple
+   *split-apply-combine* paradigm that we originally prototyped in
+   [PMDA][] [^PMDApaper]. What's really exciting is that *any* analysis code
+   that is based on [MDAnalysis.analysis.base.AnalysisBase][] can
+   enable parallelization with a few lines of extra code---all the
+   hard work is done behind the scenes in the base class (in a way
+   that is fully backwards compatible!).
+
+   This new feature is the work of @marinegor who brought his
+   [GSoC 2023 project][] to completion, with great contributions by
+   @p-j-smith, @yuxuanzhuang and @RMeli [^parallelizationPR].
+   
+   Not all MDAnalysis analysis classes have parallelization enabled
+   yet but @talagayev has been working tirelessly on already updating
+   `GNMAnalysis`, `BAT`, `Dihdedral`, `Ramachandran`, `Janin`, `DSSP`
+   (yes, MDAnalysis has _finally_ got DSSP, based on [pydssp][], also
+   thanks to @marinegor), `HydrogenBondAnalysis`, in addition to
+   `RMSD`. 
+
+
+## License change to LGPL
```

Comment:
This was a team effort but it would be good to highlight your and I publicly here @orbeckst (at least I would like it).

Reply:
ok, will add